lua-api-crates/plugin/src/lib.rs

```diff
@@ -12,6 +12,7 @@ use wezterm_dynamic::{FromDynamic, ToDynamic};
 struct RepoSpec {
     url: String,
     component: String,
+    plugin_dir: String,
```

Comment:
Let's use PathBuf here so that we can centralize the path -> string conversion into one place (the ToDynamic trait) and keep the code in here simpler

```suggestion
    plugin_dir: PathBuf,
```

Reply:
good idea, that is a lot more clean